server/src/pages/settings/view.js

```diff
@@ -75,20 +75,12 @@ class Body extends React.Component {
             <Localized id="settingsGuestAccountMessage">
               <p className="title">Guest Account</p>
             </Localized>
-            <Localized id="settingsSignInInvite">
-              <p className="info">Sign in to sync across devices</p>
-            </Localized>
```

Comment:
It doesn't seem right to me that we are removing these...? Don't we still want to display these messages to people who aren't logged in via FxA?

Reply:
We are redirecting user to homepage when a user doesn't have accountId cookie set. 

This [block of code](https://github.com/mozilla-services/screenshots/blob/55a4a35b79860f7c066882f7199879d1d82a1c32/server/src/pages/settings/view.js#L71) shows only if accountInfo is null that is querying db for the accountInfo doesn't return any record. 
If needed for this edge case, we can keep the strings with word sync removed from messaging.